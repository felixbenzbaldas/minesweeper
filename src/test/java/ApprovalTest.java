import org.approvaltests.Approvals;
import org.approvaltests.reporters.DiffReporter;
import org.approvaltests.reporters.UseReporter;
import org.example.App;
import org.junit.jupiter.api.Test;

@UseReporter(DiffReporter.class)
public class ApprovalTest {

    @Test
    public void testFieldWithExactlyOneMine() {
        Approvals.verify(App.getOutput("*"));
    }

    @Test
    public void testFieldWithNoMine() {
        Approvals.verify(App.getOutput("."));
    }

//    @Test
//    public void testBoardWithTwoFields_oneMine() {
//        Approvals.verify(App.getOutput("*."));
//    }


}
