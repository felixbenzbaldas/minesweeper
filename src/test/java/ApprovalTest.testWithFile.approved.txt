1*21001*211*21011100000
12*22245*212*101*332100
012*3****52211023***100
0012*5*8***10001*6*5211
000113****3100012***11*